modules/performance.nix

```diff
@@ -60,6 +60,12 @@ in
       luaLib = lib.mkEnableOption "luaLib" // {
         description = "Whether to byte compile lua library.";
       };
+      excludePlugins = lib.mkOption {
```

Comment:
Nit: "exclude plugins" sounds like a boolean option; whether to exclude plugins. As this is a list of plugins to be excluded, I think "excluded plugins" is more natural.